content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -17,6 +17,15 @@ You need [`read and write variables` permissions](/docs/cloud/users-teams-organi
 - The Variables API for [workspace-specific variables](/docs/cloud/api/workspace-variables.html) and [variable sets](/docs/cloud/api/variable-sets.html).
 - The `tfe` provider's [`tfe_variable`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/variable) resource, which can be more convenient for bulk management.
 
+## Run-Specific Variables
```

Comment:
Could we add a line in the page intro (right after the first sentence) that says "You can also set variable values specifically for each run on the command line."